jina/helper.py

```diff
@@ -21,6 +21,9 @@
 
 import numpy as np
 
+if False:
```

Comment:
why is this needed'

Reply:
Ah it was from when the `filter_keys` was here. Removing the import left a `pass` behind it seems